VideoLocker/res/values/strings.xml

```diff
@@ -23,252 +28,401 @@
+    <!-- Header on study groups screen encouraging users to create a group -->
     <string name="empty_groups_header">More minds are better than one</string>
+    <!-- Message on study groups screen encouraging users to create a group -->
     <string name="empty_groups_msg">Form a study group with your friends to stay motivated, learn more, and conquer your courses!</string>
+    <!-- Message on study groups screen indicating requirement -->
     <string name="empty_groups_msg2">You must connect to a Facebook account to use this feature</string>
+    <!-- Message shown after creating a study group -->
     <string name="group_created">Group Created</string>
+    <!-- Message shown after inviting friends to a study group -->
     <string name="friends_invited">Friends Invited</string>
-    <string name="groups_facebook_disconnected">You must connect a facebook account to use this feature\n</string>
+    <!-- Button title for action that allows users to share this app -->
     <string name="btn_share_app">Tell your friends about edX</string>
+    <!-- Header for social friends list -->
     <string name="friends_with_edx_accounts">Friends with edX accounts</string>
+    <!-- Header for friends search -->
     <string name="search_results">Search results</string>
+    <!-- Button allowing a user to search for friends -->
     <string name="friends_search">Search</string>
+    <!-- Button text to complete an action -->
     <string name="done_text">Done</string>
+    <!-- Label indicating unread actions. Argument is a count -->
     <string name="unread_text">%s Unread</string>
+    <!-- Label indicating number of members in a study group Argument is a count -->
     <string name="group_summary_count">Group Members (%1$s)</string>
+    <!-- Label indicating a user has no friends in a study group -->
     <string name="group_summary_no_friends">You do not have any friends in this group</string>
+    <!-- Alert dialog title when a user without facebook performs an action that requires Facebook -->
     <string name="dialog_fbinstall_title">Facebook Required</string>
+    <!-- Alert dialog message when a user without facebook performs an action that requires Facebook -->
     <string name="dialog_fbinstall_msg">That feature requires the facebook app to be installed</string>
+    <!-- Alert dialog button action to open up the google play store -->
     <string name="dialog_fbinstall_positive">Show in Google Play</string>
 
     <!-- My Groups -->
 
+    <!-- Button label to bring up course finder -->
     <string name="find_course_text">Looking for a new challenge?</string>
+    <!-- Button label to bring up course finder -->
     <string name="find_course_btn_text">Find a mobile-friendly course</string>
-    <string name="signin_with_text">Or Sign in with</string>
+    <!-- Button label to bring up course finder -->
+    <string name="signin_with_text">signing in to this app in with</string>
```

Comment:
No that was a mistake. Thanks for catching it. You'll note that the comment is wrong too.